extension/src/services/audio-recorder-service.ts

```diff
@@ -175,9 +177,10 @@ export default class AudioRecorderService {
 
     private _prepareForAudioDataResponse(requestId: string): Promise<string> {
         if (this.audioBase64Promise !== undefined) {
-            this.audioBase64Reject!(
-                new Error('New audio recording request received before the current one could finish')
-            );
+            // During bulk export, a new audio request can arrive while a previous one
```

Comment:
Rather than resolving with an empty string, which would make "empty string" a possible return value that should be handled by all callers, can we keep the `reject` with an exception that can be optionally handled at the judgment of callers? The exception can be a custom one distinguishable from others.